src/core/initialState.js

```diff
@@ -40,6 +40,7 @@ export default {
   textProperties: {
     next: 'Next',
     previous: 'Previous',
-    settingsToggle: 'Settings'
+    settingsToggle: 'Settings',
+    filterPlaceHolder: 'Filter'
```

Comment:
1. Can we rename this as `filterPlaceholder` since the attribute is `placeholder` not `placeHolder`?
2. Maybe keep the list alphabetized?